pass/ProteusPass.cpp

```diff
@@ -250,10 +252,11 @@ class ProteusJitPassImpl {
       if (isDeviceCompilation(M)) {
         ModuleDeviceKernels = getDeviceKernels(M);
         if (!isDeviceKernel(Fn))
-          FATAL_ERROR(std::string{} + __FILE__ + ":" +
-                      std::to_string(__LINE__) +
-                      " => Expected the annotated Fn " + Fn->getName() +
-                      " to be a kernel function!");
+          return;
```

Comment:
Why `return`? Shouldn't it be continue. There may other JIT kernels in the module.